tangelo/toolboxes/molecular_computation/mm_charges_solver.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2023 Good Chemistry Company.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import abc
+from typing import List, Tuple, Union, Type
+
+from tangelo.helpers.utils import is_package_installed
+from tangelo.problem_decomposition.oniom._helpers.helper_classes import Fragment
+from tangelo.toolboxes.molecular_computation.molecule import atom_string_to_list
+
+
+class MMChargesSolver(abc.ABC):
+    """Instantiate electrostatic charges solver"""
+    def __init__(self):
+        pass
+
+    @abc.abstractmethod
+    def get_charges(self) -> List[Tuple[float, Tuple[float, float, float]]]:
+        """Obtain the charges for the given low parameters of the input fragment (e.g. filenames, list(s) of coordinates)
+
+        Returns:
+            List[float]: The list of charges
+            List[Tuple(float, Tuple(float, float, float))]: The atoms and geometries of the charges"""
+        pass
+
+
+def convert_files_to_pdbs(filenames: Union[str, List[str]]):
+    """Convert file or list of files to pdb files using openbabel.
+
+    Args:
+        filename (Union[str, List[str]]): The filename(s) that describe the MM region
```

Comment:
name of argument is "filenames" according to signature, and it's gonna be written "file name(s)" in your sentence afterwards